text/0024-scheduled-updates.md

```diff
@@ -19,104 +19,109 @@ As of this writing, all of my datasets are stale. They haven't been updated beca
+Building a fog service requires special consideration to get right. This feature proposes running parts of Qri as a background process on a user's primary computing device. In this setting, we need to take great care to avoid interrupting a user's day-to-day computing experience, while also handling changing connectivity & resource availability. There is a very real chance that as a consequence of an update a user scheulded 6 months ago, Qri will be slated to start an intense script on a laptop with 5% battery left at a coffee shop while connected to restricted Wifi, while the user is nervously trying to get some dockerized python script to process a report that's due in an hour. We should aim to shape both the technical foundations and user experince of Qri to do the right thing in that context.
 
 This is a very high bar, but I think it's the standard we should work toward. Qri cron is both a big first step and a chance to iterate on meeting these requirements before including more parts of the Qri daemon in this "fog service" paradigm.
 
 # Guide-level explanation
 [guide-level-explanation]: #guide-level-explanation
 
-Here's a typical workflow for qri cron using datasets. This example shows working with the command line. As usual all of this would also be accessible over the local API. To start with, `qri cron` is a command with a number of subcommands. Ths first is `qri cron add`:
+Here's a typical workflow for qri cron using datasets. This example shows working with the command line. As usual all of this would also be accessible over the local API. This change proposes breaking the current `qri update` API, replacing the current update command with a number of subcommands nested under `qri update`. 
+
+The previous `qri update` command is an alias for `qri save --recall tf`. The english phrasing: "re-run that transform script and create a new version if something changes". This this command is now moved to `qri update run`. `run` is now like the refresh button on a web browser, manually invoking an update outside of it's scheduled execution time. Users should only need `qri update run` when they want to override the new default behaviour of having scheduled updates execute automatically.
+
+Moving on to the new stuff. The first is `qri update schedule`:
 
 ```
-$ qri cron add me/dataset_name
+$ qri update schedule me/dataset_name
 scheduled me/dataset_name
 updates every 2 days
 next update: Tues. April 11, 11:30:00
 
-WARNING: cron daemon is not currently running. start with:
-$ qri cron start
-to start the qri cron service. until then datasets will not auto-update.
+WARNING: qri update is not currently running. start with:
+$ qri update start
```

Comment:
I like all of the new names, except for `qri update start`. I don't think it strongly enough communicates that its launching a long-lived operation. With services, like in systemd and launchd, `<some-service> start` is obviously long-lived because it's talking just about a service. But `update` does many different things. `qri update start` is very close to `qri update run`. I guess what I'm getting at is that "service" should be somewhere in the name, like `qri update start-service'.

Reply:
agreed. I'm thinking `qri update service start`